app/controllers/concerns/pagination_data.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module PaginationData
+  extend ActiveSupport::Concern
+
+  def pagination_data(objects, default_per_page: nil)
```

Comment:
I think we can remove the `default_per_page` argument from the interface here as well, and replace it with a method like this:

```ruby
private

def default_per_page
  return unless defined? DEFAULT_PER_PAGE

  DEFAULT_PER_PAGE
end
```

As this concern is included in controllers that may or may not define `DEFAULT_PER_PAGE`, we can access it directly from within the concern. So we don't need to pass it as an argument to the `pagination_data` method. What do you think?

Reply:
it would be a great improvement, making the pagination_data function call a lot more cleaner